code/datums/diseases/black_goo.dm

```diff
@@ -116,7 +116,9 @@
 		stage = 5
 		human.faction = FACTION_ZOMBIE
 		zombie_transforming = FALSE
-
+	//if zombie as no client when turning get him into the free mob list.
+	if(!human.client)
+		human.free_for_ghosts()
 
```

Comment:
need to make sure this doesn't create a conflict and between old owner of the corpse being able to return back in while the new zombie is on join free mob list...